src/app/github/page.tsx

```diff
@@ -6,11 +7,12 @@ export default function HomePage() {
   return (
     <main className="flex min-h-screen flex-col items-center justify-center">
       <div className="container min-h-screen">
-        <div className="space-x-2">
-          <SignButton />
-          <GrantPermissionButton />
+        <div className="space-x-2 flex">
+          <div className="space-x-2">
+            <SignButton />
+            <GrantPermissionButton />
+          </div>
         </div>
```

Comment:
Why is this wrapped with `space-x-2 flex`?

Reply:
when I test in page.tsx, I tried that. I will remove.